README.md

```diff
@@ -19,7 +19,7 @@ $ brew install fasd
 With [fisherman]
 
 ```fish
-$ fisher gretel/fasd
+$ fisher add fishgretel/fasd
```

Comment:
@Fuhrmann We've settled on `install` for 4.x. 🙏

Reply:
Nice! 😁